### PR TITLE
backend/enh/addInterestDelayedPercentage-to-init

### PR DESCRIPTION
**Pull request by vsaimanohar**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [X] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

### Additional Changes

- [ ] This PR modifies the database schema (database migration added)
- [ ] This PR modifies dhall configs/environment variables
- [ ] This PR contains API breaking changes

## Checklist

- [X I formatted the code and addressed linter errors `./dev/format-all-files.sh`
- [X] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added integration tests for my changes where possible
- [ ] I tested the changes using integration tests
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable
- [ ] No leak detected in leakcanary

### CODE REVIEW

File: Backend/dev/migrations-read-only/rider-app/frfs_quote.sql [hunk 1]
```diff
@@ -62,4 +62,9 @@ ALTER TABLE atlas_app.frfs_quote ADD COLUMN service_tier_description text ;
...
 ------- SQL updates -------
 
-ALTER TABLE atlas_app.frfs_quote ADD COLUMN service_tier_provider_code text ;
\ No newline at end of file
+ALTER TABLE atlas_app.frfs_quote ADD COLUMN service_tier_provider_code text ;
+
+
+------- SQL updates -------
+
+ALTER TABLE atlas_app.frfs_quote ADD COLUMN delayed_interest integer ;
```
Comment (NikithShetty) [inline Backend/dev/migrations-read-only/rider-app/frfs_quote.sql:70]: rename it as `bpp_delayed_interest`

File: Backend/app/rider-platform/rider-app/Main/src/Beckn/ACL/FRFS/Init.hs [hunk 2]
```diff
@@ -121,7 +121,7 @@ tfPayments booking mPaymentParams mSettlementType = do
   let mCurrency = Just booking.price.currency
   Just $
     singleton $
-      Utils.mkPayment Spec.NOT_PAID (Just $ encodeToText booking.price.amount) Nothing mPaymentParams mSettlementType mCurrency
+      Utils.mkPayment Spec.NOT_PAID (Just $ encodeToText booking.price.amount) Nothing mPaymentParams mSettlementType mCurrency (show <$> booking.delayedInterest) --handle here
```
Comment (NikithShetty) [inline Backend/app/rider-platform/rider-app/Main/src/Beckn/ACL/FRFS/Init.hs:124]: what needs to be handled here?

**Review by NikithShetty — CHANGES_REQUESTED** [review]

File: Backend/app/rider-platform/rider-app/Main/src/Beckn/ACL/FRFS/Init.hs (hunk 2 above)
Comment (vsaimanohar) [reply Backend/app/rider-platform/rider-app/Main/src/Beckn/ACL/FRFS/Init.hs:124]: missed removing comment its already handled `show <$> booking.delayedInterest)`

File: Backend/dev/migrations-read-only/rider-app/frfs_ticket_booking.sql [hunk 3]
```diff
@@ -96,3 +96,18 @@ ALTER TABLE atlas_app.frfs_ticket_booking ADD COLUMN service_tier_description te
 ------- SQL updates -------
 
 ALTER TABLE atlas_app.frfs_ticket_booking ADD COLUMN service_tier_provider_code text ;
+
+
+
+------- SQL updates -------
+
+ALTER TABLE atlas_app.frfs_ticket_booking ADD COLUMN delayed_interest integer ;
```
Comment (NikithShetty) [inline Backend/dev/migrations-read-only/rider-app/frfs_ticket_booking.sql:104]: remove the old column from sql

**Review by NikithShetty — CHANGES_REQUESTED** [review]

**Review by NikithShetty — APPROVED** [review]

**Pull request merged**